ports/renesas-ra/machine_uart.c

```diff
@@ -409,6 +409,14 @@ STATIC mp_obj_t machine_uart_sendbreak(mp_obj_t self_in) {
 }
 STATIC MP_DEFINE_CONST_FUN_OBJ_1(machine_uart_sendbreak_obj, machine_uart_sendbreak);
 
+/// \method uart.txdone()
+/// Return `True` if all characters have been sent.
```

Comment:
I know it's a bit consistent in this file, but please use only double slash `//` to start these comments.